README.rst

```diff
@@ -49,6 +54,14 @@ Version".
 Development version
 ===================
 
+<<<<<<< HEAD
+=======
+Check your python version >= 3.6, and install pyqt5, as instructed above in the
+`Getting started`_ section above or `Running from source on old Linux`_ section below.
+
+If you are on macOS, see the `Running from source on macOS`_ section below.
+
+>>>>>>> 6ff77e7f4... README - add instructions for pyenv (#1696)
```

Comment:
Is this a diff artifact that made it in? I'm confused...

Reply:
Yea, I just fixed this.